src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,37 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/info")
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
     //TODO: implement
+    
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(LocalDateTime.now().toString());
+        req.getRequestDispatcher("/WEB-INF/views/info.jsp").forward(req, resp);
+        
+    }
+    
+    @Override
+    public void init() throws ServletException {
+        System.out.println("MyResource was initiated");
+        super.init();
```

Comment:
according to doc.oracle (https://docs.oracle.com/javaee/5/api/javax/servlet/GenericServlet.html#init%28%29)
if we override the init() method without parameters, we don't need  super.init(), it will be called any way and with config parameters by servlet container. It was confirmed today on Q&A

Reply:
ty for repeating, apparently one time was not enough for me (no sarcasm)